pkg/login/social/generic_oauth.go

```diff
@@ -275,6 +314,7 @@ func (s *SocialGenericOAuth) extractEmail(data *UserInfoJson) string {
 		if emailErr == nil {
 			return emailAddr.Address
 		}
+		s.log.Debug("Failed to parse e-mail address", "email", emailAddr, "err", emailErr.Error())
```

Comment:
Think there's a chance of `emailAddr` being nil here so would suggest removing `"email", emailAddr` from the log statement.